pytket/tests/quipper_test.py

```diff
@@ -17,10 +17,9 @@
 import pytest  # type: ignore
 from pytket import Circuit
 from pytket.quipper import circuit_from_quipper
-from pytket.transform import Transform  # type: ignore
+from pytket.transform import Transform
 from pytket.utils.results import compare_unitaries
 import numpy as np
-import os
```

Comment:
Not sure, it maybe still wouldn't. My IDE often gives suggestions for code improvement such as removing unused imports or making a variable a const ref or using emplace_back instead of push_back. And often I do them.